.github/actions/run-interop-ping-test/action.yml

```
@@ -34,7 +34,7 @@ runs:
       shell: bash
 
     - name: Configure AWS credentials for S3 build cache
-      if: ${{ inputs.s3-access-key-id }} != "" && ${{ inputs.s3-secret-access-key }} != ""
+      if: inputs.s3-access-key-id != '' && inputs.s3-secret-access-key != ''
```

GitHub actions syntax is so subtle it is really annoying. Maybe I should pick up my idea of writing a linter for this again ...

My first guess at what is wrong here would have been:


```suggestion
      if: ${{ inputs.s3-access-key-id != '' }} && ${{ inputs.s3-secret-access-key != '' }}
```